vm-setup/monitoring/prometheus/monitoring-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: monitoring
+  labels:
+    name: monitoring
```

Comment:
New line is missing at EOF

Reply:
I did remove the whole file, since we are deploying namespace [here](https://github.com/metal3-io/metal3-dev-env/blob/cb68a92ee148ded537727cec9a207c1064b5f962/03_launch_mgmt_cluster.sh#L501)